MaintenanceManager/MaintenanceManager.cpp

```diff
@@ -73,6 +73,7 @@ using namespace std;
 #define PROC_DIR "/proc"
 #define TR181_AUTOREBOOT_ENABLE "Device.DeviceInfo.X_RDKCENTRAL-COM_RFC.Feature.AutoReboot.Enable"
 #define TR181_STOP_MAINTENANCE  "Device.DeviceInfo.X_RDKCENTRAL-COM_RFC.Feature.StopMaintenance.Enable"
+#define TR181_RDKVFWUPGRADER  "Device.DeviceInfo.X_RDKCENTRAL-COM_RFC.Feature.RDKFirmwareUpgrader.Enable"
```

Comment:
RFC check not needed in MM

Reply:
This RFC check is required because we have firmware download via script method and C method. This rfc is use for deciding either download is via C or script.

Reply:
This RFC check is required because we have firmware download via script method and C method. This rfc is use for deciding either download is via C or script.